roles/azimuth_caas_operator/defaults/main.yaml

```diff
@@ -0,0 +1,280 @@
+  gitVersion: "{{ azimuth_caas_stackhpc_slurm_appliance_git_version }}"
+  uiMetaUrl: "{{ azimuth_caas_stackhpc_slurm_appliance_metadata_url }}"
+  playbook: "{{ azimuth_caas_stackhpc_slurm_appliance_playbook }}"
+  extraVars: "{{ azimuth_caas_stackhpc_slurm_appliance_extra_vars }}"
+
+
+# Indicates if the StackHPC workstation should be enabled
+azimuth_caas_stackhpc_workstation_enabled: "{{ azimuth_clusters_enabled }}"
+# The git URL for the StackHPC workstation
+azimuth_caas_stackhpc_workstation_git_url: https://github.com/stackhpc/caas-workstation.git
+# The git version for the StackHPC workstation
+azimuth_caas_stackhpc_workstation_git_version: 8c86cfebb02ecd8a9ec2d4dccf806aa5989670f2
+# The playbook to use for the workstation appliance
+azimuth_caas_stackhpc_workstation_playbook: workstation.yml
+# The metadata URL for the StackHPC workstation
+azimuth_caas_stackhpc_workstation_metadata_url: >-
+  https://raw.githubusercontent.com/stackhpc/caas-workstation/{{ azimuth_caas_stackhpc_workstation_git_version }}/ui-meta/workstation.yml
+# The ID of the image for use with the StackHPC workstation
+# By default, use the workstation image from community images if available
+azimuth_caas_stackhpc_workstation_image: >-
+  {{-
+    (
+      community_images_image_ids.workstation
+      if (
+        community_images_image_ids is defined and
+        'workstation' in community_images_image_ids
+      )
+      else undef(hint = 'azimuth_caas_stackhpc_workstation_image is required')
+    )
+    if azimuth_caas_stackhpc_workstation_enabled
+    else None
+  }}
+# The default extra vars for templates in the StackHPC workstation
+azimuth_caas_stackhpc_workstation_extra_vars_defaults:
+  cluster_image: "{{ azimuth_caas_stackhpc_workstation_image }}"
+# extra_vars overrides for templates in the StackHPC workstation
+azimuth_caas_stackhpc_workstation_extra_vars_overrides: {}
+azimuth_caas_stackhpc_workstation_extra_vars: >-
+  {{-
+    azimuth_caas_stackhpc_workstation_extra_vars_defaults |
+      combine(azimuth_caas_stackhpc_workstation_extra_vars_overrides)
+  }}
+# The template definition for the StackHPC workstation appliance
+azimuth_caas_stackhpc_workstation_template:
+  gitUrl: "{{ azimuth_caas_stackhpc_workstation_git_url }}"
+  gitVersion: "{{ azimuth_caas_stackhpc_workstation_git_version }}"
+  uiMetaUrl: "{{ azimuth_caas_stackhpc_workstation_metadata_url }}"
+  playbook: "{{ azimuth_caas_stackhpc_workstation_playbook }}"
+  extraVars: "{{ azimuth_caas_stackhpc_workstation_extra_vars }}"
+
+
+# Indicates if the StackHPC repo2docker appliance should be enabled
+azimuth_caas_stackhpc_repo2docker_enabled: "{{ azimuth_clusters_enabled }}"
+# The git URL for the StackHPC repo2docker appliance
+azimuth_caas_stackhpc_repo2docker_git_url: https://github.com/stackhpc/caas-repo2docker.git
+# The git version for the StackHPC repo2docker appliance
+azimuth_caas_stackhpc_repo2docker_git_version: 8385160f69646e615a51173e107e20e2485e2457
```

Comment:
These should be tags, but we have a task to go fix that I guess.

Reply:
That will happen as part of the release